src/rendering/renderers/shared/system/AbstractRenderer.ts

```diff
@@ -316,4 +318,29 @@ export class AbstractRenderer<PIPES, OPTIONS>
         writeable.renderPipes = null;
         writeable.runners = null;
     }
+
+    /**
+     * @deprecated since 8.0.0
+     * @param container - the container to render
+     * @param options - options to use when generating the texture
+     * @returns a texture
+     */
+    public generateTexture(container: Container, options?: IGenerateTextureOptions): Texture
+    {
+        deprecation(
+            v8_0_0,
+            'renderer.generateTexture() is deprecated, please use renderer.textureGenerator.generateTexture() directly'
+        );
+
+        return this._generateTexture(container, options);
```

Comment:
lets replace this with `renderer.textureGenerator.generateTexture({
    target:container,
    ...options
})`